stdlib/distutils/log.pyi

```diff
@@ -7,7 +8,11 @@ ERROR: int
 FATAL: int
 
 class Log:
-    def __init__(self, threshold: int = ...) -> None: ...
+    if sys.version_info >= (3, 9):
+        def __init__(self, threshold: int = 3) -> None: ...
+    else:
+        def __init__(self, threshold: int = 30) -> None: ...
```

Comment:
Not sure what's going on but it looks to me like the default is also 3 (=`WARN`) in 3.7: https://github.com/python/cpython/blob/d729c5c9d49df41d5986be7507b98cbec1b7349e/Lib/distutils/log.py#L16

Reply:
Really weird. This is what I get locally on 3.7:

```pycon
>>> import distutils.log
>>> help(distutils.log.Log.__init__)
Help on function __init__ in module distutils.log:

__init__(self, threshold=30)
    Initialize the logger with a name and an optional level.
```

My local 3.7 environment is using conda -- I can only assume something's wrong with my local environment somehow? Anyhow, I'll revert this one.

Reply:
Maybe stuptools is working its magic.

Reply:
Interestingly I get this too:

```
Python 3.7.16 (default, Dec 26 2022, 20:28:00) 
[Clang 14.0.0 (clang-1400.0.29.202)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import distutils.log
>>> distutils.log.Log
<class 'distutils.log.Log'>
>>> distutils.log.Log.__init__.__defaults__
(30,)
```

And it's indeed setuptools' fault:

```
>>> distutils.log
<module 'distutils.log' from '/Users/jelle/.pyenv/versions/3.7.16/lib/python3.7/site-packages/setuptools/_distutils/log.py'>
```

Reply:
> And it's indeed setuptools' fault:

classic